functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/project/ExecutorSpec.groovy

```diff
@@ -0,0 +1,217 @@
+package org.rundeck.tests.functional.selenium.project
+
+import org.rundeck.util.annotations.SeleniumCoreTest
+import org.rundeck.util.container.SeleniumBase
+import org.rundeck.util.gui.pages.home.HomePage
+import org.rundeck.util.gui.pages.login.LoginPage
+import org.rundeck.util.gui.pages.project.DashboardPage
+import org.rundeck.util.gui.pages.project.ProjectEditPage
+
+import java.util.stream.Collectors
+
+@SeleniumCoreTest
+class ExecutorSpec extends SeleniumBase {
+
+    private static final List<String> availableExecutors = Arrays.asList(new String[] { "SSH", "Local", "SSHJ-SSH", "Script Execution", "Stub", "Ansible Ad-Hoc Node Executor", "WinRM Node Executor Python", "openssh / executor" });
+
+    def "Check node executor list"(){
```

Comment:
I think this test should be marked as excluded for pro since pro has a different set of node executors